docs/oid-info.md

```diff
@@ -0,0 +1,23 @@
+# Sigstore OID information
+
+## Description
+
+Sigstore maintains its own Private Enterprise Number (57264) with the Internet
+Assigned Numbers Authority to help identify and organize additional metadata in
+code signing certificates issued by Fulcio instances. This document aims to
+provide a simple directory of values in use with an explanation of their
+meaning.
+
+## Directory
+
+Note that all values begin from the root OID 1.3.6.1.4.1.57264 [registered by
```

Comment:
You can use http://oid-info.com/get/1.3.6.1.4.1.57264

I would note that http://oid-info.com/get/1.3.6.1.4.1.57264 is not the source of truth (it's self-maintained by the owners of 57264 [me :P], not IANA), that the source of truth is in the respective repos (we can update the links once we include docs for each)

Reply:
I just registered http://oid-info.com/get/1.3.6.1.4.1.57264.3 (you won't see anything yet since the TAC and the site admin have to approve it)

Reply:
Whoops, didn't survive the copy/paste. Re-added!